src/site/content/en/blog/tabbed-application-mode/index.md

```diff
@@ -108,10 +104,75 @@ To use tabbed application mode, developers need to opt their apps in by setting
 
 ```json
 {
-  …
   "display": "standalone",
+  "display_override": ["tabbed"]
+}
+```
+
+Next, the property `"tab_strip"` can optionally be used to fine-tune the tab behavior. It has two
+allowed sub-properties, `"home_tab"` and `"new_tab_button"`. If the `"tab_strip"` property is not
+present, the particular properties' `"auto"` values are used. The browser can decide what values to
+use for `"auto"`.
+
+### Home tab
+
+The home tab is a pinned tab that, if enabled for an app, should always be present when the app is
+open. This tab should never navigate. Links clicked from this tab should open in a new app tab. Apps
+can choose to customize the URL this tab is locked to and the icon displayed on the tab.
+
+The allowed values for `"home_tab"` are:
+
+- `"auto"` to let the browser determine what to do.
+- `"absent"` to tell the browser to not show a home tab.
+- An object with two sub-properties:
+  - `"url"` with the allowed values of `"auto"` or a URL to lock the home tab to.
+  - `"icons"` with the allowed values of `"auto"` or an array of icons as in the main
+    [`"icons"` property](/add-manifest/#icons).
+
+### New tab button
+
+The new tab button, if present, should open a new tab at a URL that is within the scope of the app.
+The app may choose to set a custom URL and icon for this button. Browsers can decide how to handle
```

Comment:
>custom URL

Based on the previous statement, I'd think this goes without saying. Am I missing something?

Reply:
It's a different context, just the behavior happens to be the same.

Reply:
If custom URL = out of scope URL, you should make the clear.

Reply:
Added that the URL needs to be in-scope.